tests/test_compute_basic.py

```diff
@@ -328,17 +328,29 @@ def test_const_tensor_int():
     def test_kernel(dtype, size):
         hcl.init(dtype)
 
-        np_A = np.random.randint(10, size=size)
+        np_A = np.random.randint(-15, 15, size=size, dtype=np.int32)
         py_A = np_A.tolist()
 
         def kernel():
-            cp1 = hcl.const_tensor(np_A)
-            cp2 = hcl.const_tensor(py_A)
+            cp1 = hcl.const_tensor(np_A, dtype=dtype)
+            cp2 = hcl.const_tensor(py_A, dtype=dtype)
             return hcl.compute(np_A.shape, lambda *x: cp1[x] + cp2[x])
 
         O = hcl.placeholder(np_A.shape)
         s = hcl.create_schedule([], kernel)
         f = hcl.build(s)
+        # Test the generated bitwidth is the same as what user specified
+        from hcl_mlir.dialects import memref as memref_d
+        from hcl_mlir.ir import MemRefType, IntegerType, TypeAttr
+
+        for op in s.module.body.operations:
+            if isinstance(op, memref_d.GlobalOp):
+                assert (
+                    IntegerType(
+                        MemRefType(TypeAttr(op.attributes["type"]).value).element_type
+                    ).width
+                    == dtype.bits
+                )
```

Comment:
This is a great way to test the data type in the IR, can we do the same thing for FixedPoint test here? https://github.com/cornell-zhang/heterocl/blob/2094c7c7c43792bc9c7b5366d41cd97f67f6a683/tests/test_compute_basic.py#L388-L419